be/src/olap/base_compaction.cpp

```diff
@@ -81,16 +84,35 @@ Status BaseCompaction::execute_compact_impl() {
     return Status::OK();
 }
 
+void BaseCompaction::_filter_input_rowset() {
+    // if enable dup key skip big file and no delete predicate
+    // we skip big files too save resources
+    if (!config::enable_dup_key_base_compaction_skip_big_file ||
+        _tablet->keys_type() != KeysType::DUP_KEYS || _tablet->delete_predicates().size() != 0) {
+        return;
+    }
+    int64_t max_size = config::base_compaction_dup_key_max_file_size_mbytes;
```

Comment:
should be `base_compaction_dup_key_max_file_size_mbytes * 1024 * 1024`?

Reply:
fixed